src/components/CommentList/CommentList.tsx

```diff
@@ -1,5 +1,25 @@
+/* eslint-disable implicit-arrow-linebreak */
+
 import React from 'react';
+import commentsFromServer from '../../api/comments';
+import { Comment } from '../../types/Comment';
+import { CommentInfo } from '../CommentInfo';
+import './CommentList.scss';
+
+type Props = {
+  postId: number,
+};
+
+export const CommentList: React.FC<Props> = ({ postId }) => {
+  const found = (element: Comment) => element.postId === postId;
```

Comment:
Consider renaming the function `found` to a more descriptive name like `hasCommentForPost`.